sklearn/cluster/_affinity_propagation.py

```diff
@@ -84,10 +84,10 @@ def affinity_propagation(S, *, preference=None, convergence_iter=15,
     Returns
     -------
 
-    cluster_centers_indices : array, shape (n_clusters,)
+    cluster_centers_indices : array of shape (n_clusters,)
```

Comment:
For `numpy.ndarray` we use `ndarray` instead of `array`.

Reply:
ok